lib/modules/manager/pep621/extract.spec.ts

```diff
@@ -326,6 +328,68 @@ describe('modules/manager/pep621/extract', () => {
       ]);
     });
 
+    it('should skip dependencies with unsupported uv sources', async () => {
```

Comment:
please don't move test with other changes in same PR. it can easily lead to regression.

Reply:
Sure, sorry, thought it was minimal enough to not deserve a dedicated PR, especially since I initially added this test. Created a dedicated PR for this change: https://github.com/renovatebot/renovate/pull/32195.